usability_webhooks/controllers/main.py

```diff
@@ -37,25 +38,38 @@ def _create_api_logs(self, model, vals, function):
             # Error from odoo exception, rollback all data (if config in system parameter)
             if rollback_except:
                 request.env.cr.rollback()
-        request.env["api.log"].create(data_dict)
+        if vals["is_create_logs"]:
```

Comment:
```suggestion
        if vals["is_create_log"]:
```

ไม่ต้องใช้ s ทั้งหมด